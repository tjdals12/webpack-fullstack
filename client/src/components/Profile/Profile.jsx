import React from 'react';
import PropTypes from 'prop-types';

export default function Profile({ name, age }) {
    return (
        <div>
            <p>{name}</p>
            <p>{age}</p>
        </div>
    );
}

Profile.propTypes = {
    name: PropTypes.string.isRequired,
    age: PropTypes.string.isRequired,
};
