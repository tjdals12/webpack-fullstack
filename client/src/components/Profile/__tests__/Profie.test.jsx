import React from 'react';
import { render } from '@testing-library/react';
import Profile from '../Profile';

describe('<Profile />', () => {
    it('matches snapshot', () => {
        const utils = render(<Profile name="Min" age="26" />);
        expect(utils.container).toMatchSnapshot();
    });

    it('has two text', () => {
        const utils = render(<Profile name="Min" age="26" />);

        utils.getByText('Min');
        utils.getByText('26');
    });
});
