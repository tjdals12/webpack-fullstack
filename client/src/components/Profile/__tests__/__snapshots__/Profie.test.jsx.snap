// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <p>
      Min
    </p>
    <p>
      26
    </p>
  </div>
</div>
`;
